docs/tests/system_checklist.rst

```diff
@@ -175,20 +175,16 @@ Checklist
     Crontabber
     ==========
 
-    Is crontabber working?
+    Is the crontabber node working?
 
-    * Check healthcheck endpoint (/monitoring/crontabber/)
```

Comment:
Should ``/monitoring/cron/`` be mentioned here or in ``cron.rst``?

Reply:
Mmm... I think no. I want to phase it out, but I don't remember offhand where it's used if anywhere. It's much better to check the Django admin for Jobs and Logs. If we (admin) need an easier to interpret high-level view, we should add a Django admin page for it.

I think I'd nix related lines in the system checklist, too. I can do that now.

Reply:
Oh, I already did it. Yay me!